.travis.yml

```diff
@@ -3,14 +3,38 @@ language: go
 go:
   - '1.10.x'
 
+env:
+  global:
+    # Move Kubernetes config files to the appropriate place and adjust permissions.
+    - CHANGE_MINIKUBE_NONE_USER=true
+    - KUBERNETES_VERSION=v1.10.0
+
 jobs:
   include:
-    - stage: Check go files
-      script: make verify-ci
+    - stage: Verify Go formatting
+      script: make verify-gofmt
+
     - stage: Compile
       script: make compile
+
     - stage: Tests
-      script: make test
+      script: make test-ci
+
+    - stage: E2E Tests
+      script: make test-e2e
+      before_script:
+        # Download kubectl, which is a requirement for using minikube and running e2e tests.
+        - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
@soltysh @liggitt next release I'm solving my oc problems like this! :)

Reply:
I object :stuck_out_tongue_closed_eyes: